docs/release-notes.md

```diff
@@ -33,15 +33,26 @@
 
 # Release notes
 
+## Trino Gateway 7 (19 Mar 2024)
+
+[JAR file gateway-ha-7-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/7/gateway-ha-7-jar-with-dependencies.jar),
+Docker container `trinodb/trino-gateway:7`
+
+* Replace user interface with a new modern UI. ([#116](https://github.com/trinodb/trino-gateway/pull/116))
```

Comment:
Maybe say update / improve UI instead?

Reply:
Well.. the thing is that its a completely rewrite .. so its completely different .. theoretically we could even make a list of new features of the UI but given that we have no docs for it yet I wanted to avoid that